cockroachcloud/authorization.md

```diff
@@ -1,44 +1,158 @@
 ---
-title: Authorization in CockroachDB Cloud
+title: CockroachDB Cloud Access Management Overview and FAQ
 summary: Learn about the CockroachDB Cloud Authorization features and concepts
 toc: true
 docs_area: manage
 ---
 
-{{ site.data.products.db }}'s authorization model has two levels: the {{ site.data.products.db }} organization and clusters within that organization.
+This page covers the essential concepts related to access management (authorization) in {{ site.data.products.db }}. Procedures for managing access are covered in [Managing Access in {{ site.data.products.db }}](managing-access.html).
```

Comment:
Should we mention limited access on this page, coz all new roles and authz unification for service accounts has to be enabled per customer org?

Reply:
yup, adding a note, thanks!!!